_sass/minimal-mistakes/_notices.scss

```diff
@@ -45,22 +49,34 @@
   }
 
   a {
-    color: $notice-color;
+    color: mix(#fff, $notice-color, 80%);
 
     &:hover {
-      color: mix(#000, $notice-color, 40%);
+      color: mix(#fff, $notice-color, 95%);
     }
   }
 
   code {
-    background-color: mix(#fff, $notice-color, 95%)
+    padding: 0.1em 0.1em;
+    background-color: mix(#000, $notice-color, 95%);
+    color: mix(#fff, $notice-color, 95%);
+    border-radius: 4px;
+    font-size: inherit;
+    &:before, &:after {
+      letter-spacing: -0.2em;
+      content: "\00a0";
+    }
```

Comment:
To match the rest of the code blocks, I would use [this](https://github.com/momentum-mod/docs/blob/2ee6ebb8c69d1af3ca98dcd6343911a79121bf4f/_sass/minimal-mistakes/_base.scss#L167).